core/src/test/java/org/infinispan/topology/ClusterTopologyStatefulTest.java

```diff
@@ -18,15 +17,22 @@ public class ClusterTopologyStatefulTest extends AbstractStatefulCluster {
       clusterSize = 3;
    }
 
+   private Condition coordinatorHasRebalanceDisabled() {
+      return () -> cacheManagers.stream()
+            .filter(EmbeddedCacheManager::isCoordinator)
+            .map(ecm -> TestingUtil.extractGlobalComponent(ecm, ClusterTopologyManager.class))
+            .noneMatch(ClusterTopologyManager::isRebalancingEnabled);
+   }
+
    private Condition allNodesHaveRebalanceDisabled() {
-      return () -> Arrays.stream(managers())
-            .map(ecm -> TestingUtil.extractGlobalComponent(manager(0), ClusterTopologyManager.class))
+      return () -> cacheManagers.stream()
+            .map(ecm -> TestingUtil.extractGlobalComponent(ecm, ClusterTopologyManager.class))
             .noneMatch(ClusterTopologyManager::isRebalancingEnabled);
    }
 
    private Supplier<String> dumpClusterGlobalRebalanceStatus() {
-      return () -> Arrays.stream(managers())
-            .map(ecm -> Map.entry(ecm, TestingUtil.extractGlobalComponent(manager(0), ClusterTopologyManager.class)))
```

Comment:
Seems like a pretty big issue with the test, heh.

Reply:
:sob: :sob: :sob: :sob: